.circleci/config.yml

```diff
@@ -148,7 +148,16 @@ jobs:
                  echo_green "=> Package check OK"
              done
 
+             make \
+                 -f .config \
+                 -f tmp/.packagedeps \
+                 -f <(echo '$(info $(sort $(package-y) $(package-m)))'; echo -en 'a:\n\t@:') \
+              | tr ' ' '\n' >enabled-package-subdirs.txt
              for PKG in $PKGS ; do
+                 if ! grep -qE "(^|/)$PKG$" enabled-package-subdirs.txt; then
+                        echo_red "===+ Skip building $PKG"
```

Comment:
Maybe explain why we are skipping the build in the message

Reply:
Changed to `===+ Building: docker-ce skipped.  It cannot be enabled with openwrt-sdk-ath79-generic_*.Linux-x86_64.tar.xz`

See it [in action](https://circleci.com/gh/openwrt/packages/5982?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link)